src/legacy/core_plugins/kibana/public/discover/plugin.ts

```diff
@@ -89,6 +103,27 @@ export class DiscoverPlugin implements Plugin<DiscoverSetup, DiscoverStart> {
   public initializeServices?: () => Promise<{ core: CoreStart; plugins: DiscoverStartPlugins }>;
 
   setup(core: CoreSetup, plugins: DiscoverSetupPlugins): DiscoverSetup {
+    const { querySyncStateContainer, stop: stopQuerySyncStateContainer } = getQueryStateContainer(
```

Comment:
just fwd: working on improving this https://github.com/elastic/kibana/pull/56128